utils/similarity/__init__.py

```diff
@@ -0,0 +1,12 @@
+# from https://stackoverflow.com/questions/563022/whats-python-good-practice-for-importing-and-offering-optional-features
+import sys
+
+try:
+    import fastText
+except ImportError:
+    fastText = None
+
+if fastText is None:
```

Comment:
Consider moving this into the try /except block

Reply:
It could be worth to include this try except in all the import fastText from the other modules

Reply:
> Consider moving this into the try /except block

Done.

> It could be worth to include this try except in all the import fastText from the other modules

That shouldn't be necessary. This will execute during the initialization.